portfolio/src/main/webapp/script.js

```diff
@@ -27,64 +28,55 @@ function generateImageUrl() {
  * functionality but with a new, random image. 
  */
 function newImageButton() {
-
     const pennyButton = document.getElementById('penny-button');
-    const currentImgUrl = pennyButton.getAttribute("src");
+    const currentImgUrl = pennyButton.getAttribute('src');
     let newImgUrl = currentImgUrl;
 
-    //ensure new image is not the same as old one 
+    /* ensure new image is not the same as old one */ 
     while (newImgUrl === currentImgUrl) {
         newImgUrl = generateImageUrl();
     }
 
     const pennyContainer = document.getElementById('penny-container');
-    let newHTML = `<input id="penny-button" type="image" src="${newImgUrl}"\
-     onClick="newImageButton()"/>`;
+    let newHTML = `<input id="penny-button" type="image" src="${newImgUrl}"` +
+        ` onClick="newImageButton()"/>`;
     pennyContainer.innerHTML = newHTML;
 }
 
 /**
-  * Automatically fill gallery page with all available images
+  * Automatically fills gallery page with all available images
   */
 function fillGallery() {
     const imgsContainer = document.getElementById('penny-gallery-images');
```

Comment:
where possible, consider removing any abbreviations

https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers

Reply:
will remove abbreviations!